README.md

```diff
@@ -30,13 +32,31 @@ Click on "Use this Template"
 
 ![Use this Template](https://docs.github.com/assets/images/help/repository/use-this-template-button.png)
 
-Name the repository, and provide a description. We recommend using the following naming conventions:
+Name the repository, and provide a description.
+
+Depending on the plugin relationship with the OpenSearch organization we currently recommend the following naming conventions and optional follow-up checks:
+
+### Official plugins
+
+For the **official plugins** that live within the OpenSearch organization (i.e. they are included in [OpenSearch/plugins/](https://github.com/opensearch-project/OpenSearch/tree/main/plugins) or [OpenSearch/modules/](https://github.com/opensearch-project/OpenSearch/tree/main/modules) folder), and **which share the same release cycle as OpenSearch** itself:
+
 - Do not include the word `plugin` in the repo name (e.g. [job-scheduler](https://github.com/opensearch-project/job-scheduler))
 - Use lowercase repo names
 - Use spinal case for repo names (e.g. [job-scheduler](https://github.com/opensearch-project/job-scheduler))
-- do not include the word `OpenSearch` or `OpenSearch Dashboards` in the repo name
+- Do not include the word `OpenSearch` or `OpenSearch Dashboards` in the repo name
 - Provide a meaningful description, e.g. `An OpenSearch Dashboards plugin to perform real-time and historical anomaly detection on OpenSearch data`.
 
+### 3rd party plugins
+
+For the **3rd party plugins** that are maintained as an independent projects in separate git repositories **with their own release cycles** the recommended naming convention should follow the same rules as official plugins with some exceptions and a few follow-up checks:
```

Comment:
```suggestion
For the **3rd party plugins** that are maintained as independent projects in separate Github repositories **with their own release cycles** the recommended naming convention should follow the same rules as official plugins with some exceptions and a few follow-up checks:
```

Reply:
Fixed, but actually is it Github or GitHub? (I am going with the later one...)